libvirt/utils_volume.go

```diff
@@ -167,6 +167,10 @@ func isQCOW2Header(buf []byte) (bool, error) {
 	return false, nil
 }
 
+// number of download retries on non client errors (eg. 5xx)
+const maxHTTPRetries int = 3
+const retryWaitSec = 2
```

Comment:
`super small nitpick` why the 2nd var we omit the type and in the first we have the type?  Can we have some coeherence along the vars ? :cat: 

( i think also we could move this variables *inside the the `scope` of  method `Import`, since seeing them here i would think they are global and shared along others methods/functions